frontend/src/features/research/activities/list/ActivityListView.tsx

```diff
@@ -0,0 +1,127 @@
+import { TableSort } from 'components/Table/TableSort';
+import { useApiResearchFile } from 'hooks/pims-api/useApiResearchFile';
+import useIsMounted from 'hooks/useIsMounted';
+import { defaultActivityFilter, IActivityFilter } from 'interfaces/IActivityResults';
+import { orderBy } from 'lodash';
+import { Api_Activity } from 'models/api/Activity';
+import React, { useCallback } from 'react';
+
+import { ActivityFilterForm } from './ActivityFilter/ActivityFilterForm';
+import { ActivityResults } from './ActivityResults/ActivityResults';
+import { AddActivityForm } from './ActivityResults/AddActivityForm';
+import * as Styled from './styles';
+
+export interface IActivityListViewProps {
+  researchFileId?: number;
+  hideFilters?: boolean;
+  defaultFilters?: IActivityFilter;
+}
+/**
+ * Page that displays Activity information.
+ */
+export const ActivityListView: React.FunctionComponent<IActivityListViewProps> = (
+  props: IActivityListViewProps,
+) => {
+  const { researchFileId, defaultFilters, hideFilters } = props;
+  const isMounted = useIsMounted();
+  const { getResearchActivities, postActivity } = useApiResearchFile();
+  const [sort, setSort] = React.useState<TableSort<Api_Activity>>({});
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [activityResults, setActivityResults] = React.useState<Api_Activity[]>([]);
+
+  const [filters, setFilters] = React.useState<IActivityFilter>(
+    defaultFilters ?? defaultActivityFilter,
+  );
+
+  const fetchActivities = useCallback(async () => {
+    setIsLoading(true);
+    try {
+      // const data = mockActivitiesResponse();
+      // TODO Call actual api endpoint
```

Comment:
is this todo necessary? Looks like the api is being called

Reply:
not necessary